packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -511,7 +510,7 @@ $Calendar-dayPicker-margin: 10px;
 
 .wrap span:focus,
 .wrap div:focus {
-  outline: 1px solid $ms-color-themePrimary;
+  outline: $ms-color-themePrimary;
```

Comment:
Looks like this mixin is adding 1px outer border which I don't want in this case and in fact I am removing for this change.